moonshot/integrations/cli/cli.py

```diff
@@ -41,16 +44,20 @@ def default(self, statement: Statement) -> None:
         if active_session:
             user_prompt = statement.command + " " + statement
             user_prompt = user_prompt.strip()
-            send_prompt(active_session["session_id"], user_prompt)
+            manual_red_teaming(user_prompt)
             # Update chat display with response
             update_chat_display()
 
     def postcmd(self, stop, line):
         if active_session:
+            if active_session["context_strategy"]:
+                cs_prompt = f"CS: ({active_session['context_strategy']},{active_session['cs_num_of_prev_prompts']})] > "
+            else:
+                cs_prompt = f"CS: {active_session['context_strategy']}]> "
             self.prompt = (
                 f"moonshot ({active_session['session_id']}) "
                 f"[PT: {active_session['prompt_template']}, "
-                f"CS: {active_session['context_strategy']}] > "
+                f"{cs_prompt}"
```

Comment:
just checking. Dont need to put the CS: in front or something anymore?

Reply:
`cs_prompt` is a variable with the `CS:` appended in front